diff -Npru ramdisk/init.dongle.rc 3g-ramdisk/init.dongle.rc
--- ramdisk/init.dongle.rc	1969-12-31 18:00:00.000000000 -0600
+++ 3g-ramdisk/init.dongle.rc	2012-08-27 00:24:51.012865613 -0500
@@ -0,0 +1,17 @@
+on boot
+         
+on property:dev.bootcomplete=1
+        start preheated
+
+service hotplugd /system/bin/logwrapper /system/bin/hotplugd
+    class core
+
+# preheated is a symlink to hotplug. This check if the device is already connected
+service preheated /system/bin/logwrapper /system/bin/hotplugd preheated
+    oneshot     
+    disbaled
+    
+service switch_ms_to_3g /system/bin/logwrapper /system/bin/usb_modeswitch
+    class core
+    disabled
+    oneshot
\ No newline at end of file
diff -Npru ramdisk/init.rc 3g-ramdisk/init.rc
--- ramdisk/init.rc	2012-08-27 00:25:58.749201521 -0500
+++ 3g-ramdisk/init.rc	2012-08-27 00:24:51.012865613 -0500
@@ -3,7 +3,7 @@
 # IMPORTANT: Do not create world writable files or directories.
 # This is a common source of Android security bugs.
 #
-
+import /init.dongle.rc
 import /init.usb.rc
 import /init.${ro.hardware}.rc
 import /init.trace.rc
diff -Npru ramdisk/ueventd.grouper.rc 3g-ramdisk/ueventd.grouper.rc
--- ramdisk/ueventd.grouper.rc	2012-08-27 00:25:58.753201535 -0500
+++ 3g-ramdisk/ueventd.grouper.rc	2012-08-27 00:24:51.012865613 -0500
@@ -31,3 +31,4 @@
 /dev/radio      0666    system  radio
 /dev/ion        0666    system  system
 /dev/pn544      0660    nfc     nfc
+/dev/ttyUSB*    0660   root       radio
