--- android/preloaded-classes	2012-05-04 20:16:42.000000000 -0700
+++ moded/preloaded-classes	2012-05-04 23:49:56.000000000 -0700
@@ -652,6 +652,10 @@ android.provider.Settings$NameValueCache
 android.provider.Settings$NameValueTable
 android.provider.Settings$Secure
 android.provider.Settings$System
+# BEGIN privacy-added
+android.privacy.PrivacySettingsManager
+android.privacy.PrivacySettings
+# END privacy-added
 android.renderscript.RenderScript
 android.server.BluetoothA2dpService
 android.server.BluetoothEventLoop
