diff -Npru ramdisk/init.rc 3g-ramdisk/init.rc
--- ramdisk/init.rc	2012-08-27 00:25:58.749201521 -0500
+++ 3g-ramdisk/init.rc	2012-08-27 00:24:51.012865613 -0500
@@ -3,7 +3,7 @@
 # IMPORTANT: Do not create world writable files or directories.
 # This is a common source of Android security bugs.
 #
-
+import /init.dongle.rc
 import /init.${ro.hardware}.rc
 import /init.usb.rc
 import /init.trace.rc
